eZ/Publish/API/Repository/Values/ContentType/ContentTypeUpdateStruct.php

```diff
@@ -44,12 +44,19 @@ class ContentTypeUpdateStruct extends ValueObject
     public $nameSchema;
 
     /**
-     * If set the container fllag is set to this value.
+     * If set the container flag is set to this value.
      *
      * @var bool
      */
     public $isContainer;
 
+    /**
+     * Language code of this struct.
```

Comment:
Based on a chosen direction from the previous comment the purpose of this property needs to be explained better (though direction 3. implies dropping).